test/Parameter/AddressCountryTest.php

```diff
@@ -1,10 +1,10 @@
 <?php
 
-namespace CultuurNet\SearchV3\Parameter\Test;
+namespace CultuurNet\SearchV3\Parameter;
 
-use CultuurNet\SearchV3\Parameter\AddressCountry;
+use PHPUnit\Framework\TestCase;
 
-class AddressCountryTest extends \PHPUnit_Framework_TestCase
+class AddressCountryTest extends TestCase
 {
     public function testConstructor()
```

Comment:
Can get marked with `void` return type.
But I also noticed the actual and expected values are re-ordered.

Reply:
Yes, I invested less time in the tests because I had already spent a lot of time on this and having return types on the test methods etc seems less valuable

Reply:
Fixed in https://github.com/cultuurnet/search-v3/pull/43/commits/9f57e5926d3e3010287552b02f7a1bfd22884edf and  https://github.com/cultuurnet/search-v3/pull/43/commits/a38176cd628b09617ed44bd6d61606b1ab172340

Also fixed another common PHPStorm warning in the tests in https://github.com/cultuurnet/search-v3/pull/43/commits/6bdf75ca59298c5015247be1bf6bb190088664fe

Reply:
![image](https://user-images.githubusercontent.com/12606789/104590094-1c3a3c00-566b-11eb-9101-4b3f8d747683.png)

Reply:
To be fair, we could go either way but I suggest we aim for consistency in that case

Reply:
I'm okay with either way. It's easy to revert. It's consistent in this library now, but not in comparison with udb3 libraries indeed. Any preference @LucWollants?

Reply:
I have no preference. I got used to `$this->` and ignoring the warning, but that does not mean I don't wanna switch to `self::` and no longer having to ignore the warning 😁

Reply:
I've reverted it for consistency with the other libraries/projects we work on. (Had to do it with a search & replace though instead of reverting the commit, because it conflicted with the fix for the wrong order of arguments in the assertions)